arviz/tests/external_tests/test_data_pyjags.py

```diff
@@ -29,73 +43,184 @@ def verify_equality_of_numpy_values_dictionaries(
+            f"{prior_warmup_prefix}warmup_prior": ["b", "int"],
+            f"{posterior_warmup_prefix}warmup_log_likelihood": ["y"],
+        }
+
+        fails = check_multiple_attrs(test_dict, arviz_inference_data_from_pyjags_samples_dict)
+        assert not fails
+
+
+class TestDataPyJAGSWithEstimation:
+    @pytest.fixture(scope="class")
+    def jags_prior_model(self) -> pyjags.Model:
+        EIGHT_SCHOOL_PRIOR_MODEL_CODE = """
+        model {
+            mu ~ dnorm(0.0, 1.0/25)
+            tau ~ dt(0.0, 1.0/25, 1.0) T(0, )
+            for (j in 1:J) {
+                theta_tilde[j] ~ dnorm(0.0, 1.0)
+            }
+        }
+        """
+
+        prior_model = pyjags.Model(
+            code=EIGHT_SCHOOL_PRIOR_MODEL_CODE,
+            data={"J": 8},
+            chains=4,
+            threads=4,
+            chains_per_thread=1,
+        )
+
+        return prior_model
+
+    @pytest.fixture(scope="class")
+    def jags_posterior_model(
+        self, eight_schools_params: tp.Dict[str, tp.Union[int, np.ndarray]]
+    ) -> pyjags.Model:
+        EIGHT_SCHOOL_POSTERIOR_MODEL_CODE = """
+        model {
+            mu ~ dnorm(0.0, 1.0/25)
+            tau ~ dt(0.0, 1.0/25, 1.0) T(0, )
+            for (j in 1:J) {
+                theta_tilde[j] ~ dnorm(0.0, 1.0)
+                y[j] ~ dnorm(mu + tau * theta_tilde[j], 1.0/(sigma[j]^2))
+                log_like[j] = logdensity.norm(y[j], mu + tau * theta_tilde[j], 1.0/(sigma[j]^2))
+            }
+        }
+        """
+
+        posterior_model = pyjags.Model(
+            code=EIGHT_SCHOOL_POSTERIOR_MODEL_CODE,
+            data=eight_schools_params,
+            chains=4,
+            threads=4,
+            chains_per_thread=1,
+        )
+
+        return posterior_model
+
+    @pytest.fixture(scope="class")
+    def jags_prior_samples(self, jags_prior_model: pyjags.Model) -> tp.Dict[str, np.ndarray]:
+        return jags_prior_model.sample(
+            NUMBER_OF_WARMUP_SAMPLES + NUMBER_OF_POST_WARMUP_SAMPLES, vars=PARAMETERS
+        )
+
+    @pytest.fixture(scope="class")
+    def jags_posterior_samples(
+        self, jags_posterior_model: pyjags.Model
+    ) -> tp.Dict[str, np.ndarray]:
+        return jags_posterior_model.sample(
+            NUMBER_OF_WARMUP_SAMPLES + NUMBER_OF_POST_WARMUP_SAMPLES, vars=VARIABLES
+        )
+
+    @pytest.fixture()
+    def pyjags_data(
+        self,
+        jags_prior_samples: tp.Dict[str, np.ndarray],
+        jags_posterior_samples: tp.Dict[str, np.ndarray],
+    ) -> InferenceData:
+        return from_pyjags(
+            posterior=jags_posterior_samples,
+            prior=jags_prior_samples,
+            log_likelihood={"y": "log_like"},
+            save_warmup=True,
+            warmup_iterations=NUMBER_OF_WARMUP_SAMPLES,
+        )
+
+    def test_waic(self, pyjags_data):
+        waic_result = waic(pyjags_data)
+
+        assert -31.0 < waic_result.waic < -30.0
+        assert 0.75 < waic_result.p_waic < 0.90
```

Comment:
I am continuing the conversation from previous comment here. 

I do understand it makes sense to check not only the presence of the log likelihood group but its values too, however, I don't think this should be done using waic because there are several issues unrelated to `from_pyjags` that could cause the test to fail, if possible we should get the test to fail only if what they are testing for fails. To put one example, we recently modified the default scale in which waic and loo return the results, if we had been using waic to test log likelihood data we would have had to update both waic tests _and_ all converter tests. 

I'd suggest one of the following approaches:
* if random seeding is trustworthy and implemented, checking the first row or something like this aganist some hardcoded values could work
* using some simpler and not subject to change quantity like the mean
* given that we do have posterior and log likelihood samples in the inference data, use the samples in the posterior and the observed data fixture to check the first row of log likelihood values

Reply:
Thank you for the explanation. That makes sense. I removed the test for waic and am now only testing for the presence of the likelihood group via `check_multiple_attrs`.